build-go.sh

```diff
@@ -130,7 +133,7 @@ function buildWithMatrix() {
 	# build each os/arch combo
 	while read line
 	do
-		doBuild $line $gobin $output
```

Comment:
lines: 130 in old, 165 in new.

yes its part of this function. i think this is fine.